content/en/docs/configuration/acme/dns01/route53.md

```diff
@@ -180,6 +180,8 @@ spec:
 
 Note that, as mentioned above, the pod is using `arn:aws:iam::XXXXXXXXXXX:role/cert-manager` as a credentials source in Account X, but the `ClusterIssuer` ultimately assumes the `arn:aws:iam::YYYYYYYYYYYY:role/dns-manager` role to actually make changes in Route53 zones located in Account Y.
 
+When using a `ClusterIssuer` provided secret have to live inside the `kube-system` namespace so that challenges can properly fetch the secret.
```

Comment:
@MartianGreed thanks for raising the issue and apologies for not replying sooner.

Any Secret associated with a ClusterIssuer must reside in the `--cluster-resource-namespace`, 
which is the namespace `cert-manager` by default,
but may be overridden by whoever installed your cert-manager.

See  [Cluster Resource Namespace](https://cert-manager.io/docs/faq/cluster-resource/) and please help us improve that documentation if you have time.